pkg/minishift/openshift/version/openshift_versions.go

```diff
@@ -76,24 +77,12 @@ func PrintDownStreamVersions(output io.Writer, minSupportedVersion string) error
 // PrintUpStreamVersions prints the origin versions which satisfies the following conditions:
 // 	1. Major versions greater than or equal to the minimum supported and default version
 //	2. Pre-release versions greater than default version
-func PrintUpStreamVersions(output io.Writer, minSupportedVersion string, defaultVersion string) error {
-	var releaseList []string
-	data, err := getGithubReleases()
+func PrintUpStreamVersions(output io.Writer, minSupportedVersion, defaultVersion string) error {
```

Comment:
Can we actually change the `PrintUp` to be just `Print...`

Reply:
the reason why I mention this is the following:> ' `Print up` is a prhrasal verb that means to show something, while you mean to say `Upstream` and ` Downstream` which is a conjoined word (no spaces, no dash).

Reply:
@minishift/minishift-dev ^^ FYI